packages/client/src/useObservable.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect, useState } from "react";
+import { Observable } from "rxjs";
+
+// TODO: allow an optional initial value, but return type should remove the undefined if initial value provided
+//       otherwise, best to use `useObservable(...) ?? defaultValue`
+
+export const useObservable = <T>(observable: Observable<T> | null | undefined) => {
```

Comment:
> should we call this `useObservableValue`? you're probably more familiar with hook naming conventions than I am

I actually started with `useLastObservableValue` and then `useObservableValue` and then `useObservable`, because I couldn't of any other use case for putting an observable into a hook and getting something out of it. Not opposed to `useObservableValue` if you think it's clearer!

Reply:
ended up going with `useObservableValue` to match `usePromiseValue`